cypress.json

```diff
@@ -1,8 +1,10 @@
 {
   "baseUrl": "http://localhost:3000",
   "env": {
-    "API_URI": "http://localhost:8000/graphql/"
+    "API_URI": "https://pwa.demo.saleor.rocks/graphql/"
```

Comment:
Are we going to use demo graphql endpoint for tests?

Reply:
For now - yes. We dont have better environment for that right now :(